subsys/logging/Kconfig.processing

```diff
@@ -90,6 +90,17 @@ config LOG_PROCESS_THREAD_STACK_SIZE
 	help
 	  Set the internal stack size for log processing thread.
 
+config LOG_PROCESS_THREAD_SPECIFY_THREAD_PRIORITY
+	bool "Enable specifying the priority of the internal log processing thread"
+	default n
+	help
+	  When enabled the priority of the log processing thread can be set at build time.
+	  The default is to set the priority to K_LOWEST_APPLICATION_THREAD_PRIO
```

Comment:
"The default" is not clear, maybe better would be "If not enabled priority is set to ..".